setup.py

```diff
@@ -1,7 +1,8 @@
 from setuptools import setup, find_packages
+import os
 
 setup(name='u8timeseries',
-      version='0.0.2',
+      version=open("u8timeseries/VERSION", "r").read(),
```

Comment:
Is it possible to reuse tags from github? Then we don't need to add an additional file.